src/main/java/bio/terra/service/dataset/flight/DatasetWorkingMapKeys.java

```diff
@@ -12,4 +12,5 @@ private DatasetWorkingMapKeys() {}
   public static final String PROJECT_RESOURCE_ID = "projectResourceId";
   public static final String APPLICATION_DEPLOYMENT_RESOURCE_ID = "applicationDeploymentResourceId";
   public static final String STORAGE_ACCOUNT_RESOURCE_ID = "storageAccountResourceId";
+  public static final String AZURE_STORAGE_AUTH_INFO = "azureStorageAuthInfo";
```

Comment:
We now have a shared map key we should use instead: https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/common/CommonMapKeys.java#L8